lib/playgroundbook_lint/abstract_linter.rb

```diff
@@ -1,7 +1,7 @@
 require 'colored'
 require 'cork'
 
-module PlaygroundBookLint
+module Playgroundbook
```

Comment:
It came from wanting to call the executable `playgroundbook` instead of something with underscores or dashes. Might be silly, what do you think?